libraries/AP_Logger/LogFile.cpp

```diff
@@ -149,7 +149,7 @@ void AP_Logger::Write_RCIN(void)
     };
     WriteBlock(&pkt, sizeof(pkt));
 
-    const uint16_t override_mask = rc().get_override_mask();
+    const uint16_t override_mask = RC_Channels::get_override_mask();
```

Comment:
Make `get_override_mask` non-static instead?  It gets the singleton itself ATM, and it's only used on one place..